src/java.base/share/classes/jdk/internal/util/xml/impl/XMLStreamWriterImpl.java

```diff
@@ -75,7 +75,7 @@ public class XMLStreamWriterImpl implements XMLStreamWriter {
     //pretty print by default
     private boolean _doIndent = true;
     //The system line separator for writing out line breaks.
-    private char[] _lineSep = System.lineSeparator().toCharArray();
+    private final char[] _lineSep = System.lineSeparator().toCharArray();
```

Comment:
The method System.lineSeparator() is specified such that it will always return the same value. Hene, I proposes:

```
    @Stable
    private static final char[] LINE_SEP = System.lineSeparator().toCharArray();

```